src/main/java/com/market/servicemarket/util/GlobalExceptionHandler.java

```diff
@@ -12,49 +12,91 @@
         response.setResponseMessage(configurationUtil.getMessage(Constants.INVALID_FIELD_RESPONSE_CODE));
         ex.printStackTrace();
+        ex.printStackTrace(new PrintWriter(errors));
+        errors.toString();
+        try {
+            mailUtil.sendMailToAdmin(emails,errors);
+        } catch (MessagingException e) {
+            System.out.println("Unable to send exception on mail");
+        }
         return ResponseEntity.ok(response);
     }
     @Override
     protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex, HttpHeaders headers, HttpStatus status, WebRequest request) {
+        emails = configurationUtil.getMessage(Constants.ADMIN_EMAILS);
         BaseResponse response = new BaseResponse();
         response.setResponseCode(Constants.INVALID_FIELD_RESPONSE_CODE);
         response.setResponseMessage(configurationUtil.getMessage(Constants.INVALID_FIELD_RESPONSE_CODE));
         ex.printStackTrace();
+        ex.printStackTrace(new PrintWriter(errors));
+        errors.toString();
+        try {
+            mailUtil.sendMailToAdmin(emails,errors);
+        } catch (MessagingException e) {
+            System.out.println("Unable to send exception on mail");
+        }
+
         return ResponseEntity.ok(response);
     }
 
     @ExceptionHandler(RuntimeException.class)
     public ResponseEntity<Object> handleRuntimeExceptions(RuntimeException exception, WebRequest webRequest) {
+        emails = configurationUtil.getMessage(Constants.ADMIN_EMAILS);
         BaseResponse response = new BaseResponse();
         response.setResponseCode(Constants.FAILUARE_RESPNSE_CODE);
         response.setResponseMessage(configurationUtil.getMessage(Constants.FAILUARE_RESPNSE_CODE));
         ResponseEntity<Object> entity = new ResponseEntity<>(response, HttpStatus.OK);
         exception.printStackTrace();
+        exception.printStackTrace(new PrintWriter(errors));
+        errors.toString();
+        try {
+            mailUtil.sendMailToAdmin(emails,errors);
+        } catch (MessagingException e) {
+            System.out.println("Unable to send exception on mail");
+        }
         return entity;
     }
 
 
     @ExceptionHandler(Exception.class)
     public ResponseEntity<Object> handleExceptions(Exception exception, WebRequest webRequest) {
+        emails = configurationUtil.getMessage(Constants.ADMIN_EMAILS);
         BaseResponse response = new BaseResponse();
         response.setResponseCode(Constants.FAILUARE_RESPNSE_CODE);
         response.setResponseMessage(configurationUtil.getMessage(Constants.FAILUARE_RESPNSE_CODE));
         ResponseEntity<Object> entity = new ResponseEntity<>(response, HttpStatus.OK);
         exception.printStackTrace();
+        exception.printStackTrace(new PrintWriter(errors));
+        errors.toString();
+        try {
+            mailUtil.sendMailToAdmin(emails,errors);
+        } catch (MessagingException e) {
```

Comment:
send email instead of printing System.out.print, It will again try to send email .
This way, we will receive email in next try with stack trace of not snding email in first try